.github/workflows/ui-test-users-id-tasks-id.yml

```diff
@@ -0,0 +1,52 @@
+name: UI Test /users/{userid}/tasks/{taskid}
+
+on: 
+  push:
+    branches:
+      - development
+      - natalia-uitest-sprint4
+  pull_request:
+    branches: 
+      - development
+      - natalia-uitest-sprint4
```

Comment:
Probably we want to get only run the test on push because otherwise the ui won't have been deployed and thus there's nothing really to test.

Reply:
removed

Reply:
I have found this part to be very useful, actually, it deploys the changes the runs the action upon each push to the PR, otherwise no action